src/test/java/uk/gov/hmcts/ccd/domain/model/common/DisplayContextParameterUtilTest.java

```diff
@@ -0,0 +1,82 @@
+import static uk.gov.hmcts.ccd.domain.model.common.DisplayContextParameterUtil.Parameter.COLLECTION;
+import static uk.gov.hmcts.ccd.domain.model.common.DisplayContextParameterUtil.Parameter.TABLE;
+import static uk.gov.hmcts.ccd.domain.model.common.DisplayContextParameterUtil.updateDisplayContextParameter;
+
+class DisplayContextParameterUtilTest {
+
+    @Test
+    @DisplayName("should create parameter)")
+    void shouldCreateParameter() {
+        assertAll(
+            () -> assertThat(updateDisplayContextParameter("", COLLECTION, asList("allowInsert", "allowDelete")),
+                is("#COLLECTION(allowDelete,allowInsert)")),
+            () -> assertThat(updateDisplayContextParameter("", COLLECTION, asList("allowDelete")),
+                is("#COLLECTION(allowDelete)")),
+            () -> assertThat(updateDisplayContextParameter("", TABLE, asList("postcode")),
+                is("#TABLE(postcode)")),
+            () -> assertThat(updateDisplayContextParameter("", TABLE, asList("AddressLine1", "postcode")),
+                is("#TABLE(AddressLine1,postcode)"))
+                 );
+    }
+
+    @Test
+    @DisplayName("should update parameter)")
+    void shouldUpdateParameter() {
+        assertAll(
+            () -> assertThat(updateDisplayContextParameter("#COLLECTION(allowInsert,allowDelete)", COLLECTION, asList("allowInsert", "allowDelete")),
+                is("#COLLECTION(allowDelete,allowInsert)")),
+            () -> assertThat(updateDisplayContextParameter("#COLLECTION(allowInsert,allowDelete)", COLLECTION, asList("allowDelete")),
+                is("#COLLECTION(allowDelete)")),
+            () -> assertThat(updateDisplayContextParameter("#COLLECTION()", COLLECTION, asList("allowInsert")),
+                is("#COLLECTION(allowInsert)")),
+            () -> assertThat(updateDisplayContextParameter("#TABLE(param1,param2)", TABLE, asList("postcode")),
+                is("#TABLE(postcode)"))
+                 );
+    }
+
+    @Test
+    @DisplayName("should update parameter and produce multiple result)")
+    void shouldUpdateParameterMultiple() {
+        assertAll(
+            () -> assertThat(updateDisplayContextParameter("#TABLE()", COLLECTION, asList("allowInsert", "allowDelete")),
+                is("#COLLECTION(allowDelete,allowInsert),#TABLE()")),
+            () -> assertThat(updateDisplayContextParameter("#TABLE(),#COLLECTION(allowInsert,allowDelete)", COLLECTION, asList("allowDelete")),
+                is("#COLLECTION(allowDelete),#TABLE()")),
+            () -> assertThat(updateDisplayContextParameter("#TABLE(),#COLLECTION(allowInsert,allowDelete)", TABLE, asList("postcode")),
+                is("#COLLECTION(allowDelete,allowInsert),#TABLE(postcode)")),
+            () -> assertThat(updateDisplayContextParameter("#TABLE(postcode),#COLLECTION(allowInsert,allowDelete)", COLLECTION, asList("other")),
+                is("#COLLECTION(other),#TABLE(postcode)"))
+                 );
```

Comment:
mmm not sure about this but it seems to me these tests are describing a feature we don't provide. We only allow overriding of "allowDelete, allowInsert" of COLLECTION, so adding tests for all the rest seems a bit misleading to me. But we can leave them